test/test-framework/cmd/manager/main.go

```diff
@@ -140,7 +154,7 @@ func main() {
 
 // addMetrics will create the Services and Service Monitors to allow the operator export the metrics by using
 // the Prometheus operator
-func addMetrics(ctx context.Context, cfg *rest.Config, namespace string) {
```

Comment:
would a more apporiate change be to help users of cluster operators to understand how to pass this namespace in correctly rather than change the method signature for a scaffolded method?

Reply:
The value required is not obtained above.
So, IMO make no sense we pass it.

Reply:
Explained here https://github.com/operator-framework/operator-sdk/pull/2601#issuecomment-596751902 why it cannot be done.